markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+    \item Data statement:\footnote{ Description: An overall statement on the data used in the project. A spearate field is provided to list and document the origin and key characteristics of the datasets}. For external packages, the following rule:
+        \begin{itemize}
+          \item If all data is public:
+            \begin{itemize}
+            \item Fill in: "All data is public and contained in the (external)         reproducibility package."
+            \item Add data availability statement if available from journal package; often separate from the zip, has to be copied over manually.
+            \end{itemize}
+          \item If some data is confidential: Paste from the readme notes about confidentiality.
+        \end{itemize}
+    \item Datasets:\footnote{Description of this field: "This field is used to provide an itemized list of datasets used in the project. The data are not documented here (specific metadata are available for documenting data of different types, like the DDO for microdata, the ISO 19139 for geographic datasets etc)"} Not used for PP, will be used for WP, if they data sets are of proper value and stored in another WB repository (like microdata library). \textcolor{red}{No such case arose so far. TBD}
+        \begin{enumerate}
+          \item Name
+          \item Data set ID: (empty)
+          \item Note: (empty)
+          \item Access policy:
+            \begin{itemize}
+              \item Published elsewhere: Note access/license
+              \item Published with package: Authors must confirm there is no confidentiality.
+            \end{itemize}
+          \item License (empty)
+          \item License URL (empty)
+          \item Data URL
+        \end{enumerate}
+\end{enumerate}
+
+## Methods, software and scripts
+Fields that are not listed can be discarded.
+\begin{enumerate}
+  \item Methods: Econometric methods used. \textcolor{red}{This field will learn from past entries.} Possibly add notes (more details about the context)
+  \item Software: Stata 17.0, R, etc
+  \item Scripts:
+      \begin{enumerate}
+          \item File name (WP+PP): Name of the zip file. Naming convention: PrimaryID\_prg\_v01.zip
+          \item Zip package (WP+PP): Same as file name\footnote{File name and zip package are identical, because we do not list files separately, for which the field file name is there.}
+          \item Title: 
+            \begin{lstlisting}[breaklines]
+              Reproducibility package (data and code) for Title
+            \end{lstlisting}
+            Title may be shortened if adequate.
+          \item Authors: WP+PP: empty\footnote{According to descr: A list of authors and coauthors of the script} \textcolor{red}{If authors of the paper and the code differ and authors want to indicate this, add here list of authors for the code!}
+          \item Date: Year-Month (usually the completion/version date)
+          \item Software: Stata
+          \item Description: empty
```

Comment:
I would think this should be a description of what's in the zip?

Reply:
We can do that, its a decision to be made. It is though a lot of work and it is not immediately clear what is the value of this. Usually the readme has a summary anyway.

Reply:
I think it can be a simple description, like a tree of what's in the zip since people can't see it til they download (e.g. the package contains a readme file, a data folder with 5 datasets, a scripts folder with a main script and 8 sub-scripts, etc)